src/reactviews/pages/QueryResult/table/plugins/cellRangeSelector.ts

```diff
@@ -43,20 +48,25 @@ export class CellRangeSelector<T extends Slick.SlickData> implements ICellRangeS
     private decorator!: ICellRangeDecorator;
     private canvas!: HTMLCanvasElement;
     private currentlySelectedRange?: { start: Slick.Cell; end?: Slick.Cell };
+    private platform: string | undefined;
 
     public onBeforeCellRangeSelected = new Slick.Event<Slick.Cell>();
     public onCellRangeSelected = new Slick.Event<Slick.Range>();
     public onAppendCellRangeSelected = new Slick.Event<Slick.Range>();
 
-    constructor(private options: ICellRangeSelectorOptions) {
+    constructor(
+        private options: ICellRangeSelectorOptions,
+        private webViewState: VscodeWebviewContext<QueryResultWebviewState, QueryResultReducers>,
```

Comment:
why not just make platform part of the state and pass that all the way through instead of adding RPC calls and introducing complexity?

Also, dumb question, but can `process.platform` not be accessed directly in the react layer?